conf.py

```diff
@@ -118,7 +118,9 @@
     'github_user': 'coala',
     'github_repo': 'coala',
     'github_banner': True,
-    'github_type': 'star'
+    'github_type': 'star',
+    'logo': '../../../coala_logo.png',
```

Comment:
shouldn't this be put in some image directory? Then the path is probably also not that weird